### PR TITLE
Stackdriver test fix

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Extend timeout & add logging missing metrics

### CODE REVIEW

File: test/e2e/stackdriver_monitoring.go [hunk 1]
```diff
@@ -79,22 +80,38 @@ var _ = framework.KubeDescribe("Stackdriver Monitoring", func() {
...
 
-		pollingFunction := checkForMetrics(projectId, gcmService, time.Now())
-		framework.ExpectNoError(wait.Poll(pollFrequency, pollTimeout, pollingFunction))
+		metricsMap := map[string]bool{}
+		pollingFunction := checkForMetrics(projectId, gcmService, time.Now(), metricsMap)
+		err = wait.Poll(pollFrequency, pollTimeout, pollingFunction)
+		if err != nil {
+			for metric, found := range metricsMap {
+				if !found {
+					glog.Infof("Missing metric %v\n", metric)
```
Comment (piosz) [inline test/e2e/stackdriver_monitoring.go]: Can we make this a list?

File: test/e2e/stackdriver_monitoring.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ import (
 
 	"golang.org/x/oauth2/google"
 
+	"github.com/golang/glog"
```
Comment (piosz) [inline test/e2e/stackdriver_monitoring.go]: Use `framework.Logf` for logging rather than glog.

File: test/e2e/stackdriver_monitoring.go [hunk 3]
```diff
@@ -79,22 +80,38 @@ var _ = framework.KubeDescribe("Stackdriver Monitoring", func() {
 
 		rc.WaitForReplicas(replicas)
 
-		pollingFunction := checkForMetrics(projectId, gcmService, time.Now())
-		framework.ExpectNoError(wait.Poll(pollFrequency, pollTimeout, pollingFunction))
+		metricsMap := map[string]bool{}
```
Comment (piosz) [inline test/e2e/stackdriver_monitoring.go]: Can you add a TODO somewhere to make this test less noisy once debugged?

File: test/e2e/stackdriver_monitoring.go (hunk 2 above)
Comment (aleksandra-malinowska) [inline test/e2e/stackdriver_monitoring.go]: Done

**Pull request merged**